crates/rolldown_plugin_data_url/src/data_url_plugin.rs

```diff
@@ -37,7 +37,8 @@ impl Plugin for DataUrlPlugin {
           return Ok(None);
         };
         let decoded_data = if parsed.is_base64 {
-          String::from_utf8(base64_simd::STANDARD.decode_to_vec(parsed.data)?)?
```

Comment:
Would it be a problem of decoding? Is it expected to generate invalid utf8 character?

Reply:
Let me see how other bundle behave.

Reply:
We just generate almost identical css as esbulid, 
![image](https://github.com/user-attachments/assets/1843a06b-c7f6-4e5c-a83d-d13ee90b0662)
except esbuild has a prefix `\`, but webpack did not have the prefix `\`.

Reply:
rolldown generate the same dataurl string as webpack, so I think it's fine.

Reply:
As the function doc said, it returned `bytes` and did not ensure it was a valid `Utf8` string.
![image](https://github.com/user-attachments/assets/b3347363-587e-4930-aef9-b59b76faeda8)